src/PreCICE.jl

```diff
@@ -33,8 +39,23 @@ export
     getVersionInformation, actionWriteInitialData, actionWriteIterationCheckpoint, actionReadIterationCheckpoint
 
 
+@doc """
+
+    createSolverInterface(participantName::String, configFilename::String, solverProcessIndex::Integer, solverProcessSize::Integer)
```

Comment:
Is there a reason why there is indentation here and not in the documentation lines below? Is this how it is done anyway?

Reply:
Yes, in julia documentation the Signature of the function is indented to signal to julia that it is the signature and not some other text.